### ISSUE TITLE
"Eigen::aligned_allocator" for ch13/dense_RGBD/pointcloud_mapping.cpp

### ISSUE THREAD

**Issue by amiltonwong** [gaoxiang12/slambook#128]
Hi, @gaoxiang12 ,

[Line18](https://github.com/gaoxiang12/slambook/blob/master/ch13/dense_RGBD/pointcloud_mapping.cpp#L18) in `ch13/dense_RGBD/pointcloud_mapping.cpp` should be modified as : `vector<Eigen::Isometry3d, Eigen::aligned_allocator<Eigen::Isometry3d>> poses; `

**Comment by gaoxiang12** [comment]
yeah exactly, i'll change to the aligned allocator in the next version.

**Comment by siliangwu** [comment]
thanks
